newIDE/app/src/ObjectEditor/Editors/Model3DEditor.js

```diff
@@ -196,88 +280,493 @@ const Model3DEditor = ({
+                        drop={() => {
+                          moveAnimation(animationIndex);
+                        }}
+                      >
+                        {({
+                          connectDragSource,
+                          connectDropTarget,
+                          isOver,
+                          canDrop,
+                        }) =>
+                          connectDropTarget(
+                            <div
+                              key={animation.ptr}
+                              style={styles.rowContainer}
+                            >
+                              {isOver && <DropIndicator canDrop={canDrop} />}
+                              <div
+                                ref={animationRef}
+                                style={{
+                                  ...styles.rowContent,
+                                  backgroundColor:
+                                    gdevelopTheme.list.itemsBackgroundColor,
+                                }}
+                              >
+                                <Line noMargin expand alignItems="center">
+                                  {connectDragSource(
+                                    <span>
+                                      <Spacer />
+                                      <DragHandleIcon />
+                                      <Spacer />
+                                    </span>
+                                  )}
+                                  <Text noMargin noShrink>
+                                    <Trans>Animation #{animationIndex}</Trans>
+                                  </Text>
+                                  <Spacer />
+                                  <SemiControlledTextField
+                                    margin="none"
+                                    commitOnBlur
+                                    errorText={nameErrors[animation.ptr]}
+                                    translatableHintText={t`Optional animation name`}
+                                    value={animation.getName()}
+                                    onChange={text =>
+                                      changeAnimationName(animationIndex, text)
+                                    }
+                                    fullWidth
+                                  />
+                                  <IconButton
+                                    size="small"
+                                    onClick={() =>
+                                      removeAnimation(animationIndex)
+                                    }
+                                  >
+                                    <Trash />
+                                  </IconButton>
+                                </Line>
+                                <Spacer />
+                              </div>
+                              <Spacer />
+                              <ColumnStackLayout expand>
```

Comment:
I think you can use a ReponsiveLineStackLayout that will add margins (and prevent using the spacer just above) and put on the same line the select and the loop checkbox. You can maybe put a flex:1 on the select so that it does take all the available space

Reply:
Won't it be odd to have a checkbox and an unrelated field on the same line?

Reply:
True, I just found it weird to have this select that takes the whole width but maybe it's stranger to have both parameters on the same lines